public/javascripts/Admin/src/Admin.GSVLabelView.js

```diff
@@ -388,6 +434,53 @@ function AdminGSVLabelView(admin, source) {
         currButton.css('color', 'white');
     }
 
+    /**
+     * Sets the new state of a flag for the current label's audit task.
+     * @param flag
+     * @param state
+     * @private
+     */
+    function _setFlag(flag, state) {
+        let data = {
+            auditTaskId: self.taskID,
+            flag: flag,
+            state: state
+        };
+
+        // Submit the new flag state via PUT request.
+        $.ajax({
+            async: true,
+            contentType: 'application/json; charset=utf-8',
+            url: "/adminapi/setTaskFlag",
+            type: 'PUT',
+            data: JSON.stringify(data),
+            dataType: 'json',
+            success: function (result) {
+                self.flags[flag] = state;
+                _updateFlagButton();
+            },
+            error: function(xhr, textStatus, error){
+                console.error(xhr.statusText);
+                console.error(textStatus);
+                console.error(error);
+            }
+        });
+    }
+
+    /**
+     * Updates the background of each flag button depending on the flag's state
+     * @private
+     */
+    function _updateFlagButton() {
+        for (var button in self.flagButtons) {
+            if (self.flags[button]) {
+                self.flagButtons[button].css("background-color", "red");
```

Comment:
I think it's probably best if we keep them as gray like the validation buttons above. The red is a bit intense!